src/cli/lazy.py

```
@@ -0,0 +1,91 @@
+    "dedupe": ("cli.dedupe_v2", "dedupe_app", "Find and manage duplicate files."),
+    "rules": ("cli.rules", "rules_app", "Manage automated organization rules."),
+    "setup": ("cli.setup", "setup_app", "Initial configuration wizard."),
+    "suggest": ("cli.suggest", "suggest_app", "Get AI suggestions for files."),
+    "update": ("cli.update", "update_app", "Update the application."),
+}
+
+
+class LazyCommandProxy(click.Group):
+    """A proxy for a click Group that defers importing its module."""
+
+    def __init__(self, name: str, module_name: str, attr_name: str, help_text: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)